api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -342,13 +340,19 @@ private CohortReview initializeAndSaveCohortReview(String workspaceNamespace,
                                                                               Map<String, Integer> rm) {
         List<ParticipantCohortStatus> participantCohortStatuses = new ArrayList<>();
         for (List<FieldValue> row : result.iterateAll()) {
+            String birthDateTimeString = bigQueryService.getString(row, rm.get("birth_datetime"));
```

Comment:
Need to do a null / empty check here? (I expect every participant should have a birth date, but I wouldn't bake it into this code...)

Also seems like we should have a method for reading dates / datetimes from BigQuery. I'm surprised that this works / returns an epoch milli.